Gemfile

```diff
@@ -0,0 +1,6 @@
+source 'https://rubygems.org/'
+
+gem 'pmdtester', '~> 1.0.0.pre.beta2'
```

Comment:
Cool, that means, that we don't need to update PMD if we have released a new version of pmdtester (at least, once we have the first version released, we can specify here `~> 1.0` or even `~> 1`.